vscode-dotnet-runtime-library/src/Acquisition/LinuxVersionResolver.ts

```diff
@@ -218,6 +222,8 @@ Or, install Red Hat Enterprise Linux 8.0 or Red Hat Enterprise Linux 9.0 from ht
                     throw unsupportedRhelErr.error;
                 }
                 return new RedHatDistroSDKProvider(distroAndVersion, this.workerContext, this.utilityContext);
+            case 'Debian':
```

Comment:
This probably needs to be updated to the other string as well.

Reply:
`"Debian GNU/Linux"`